### ISSUE TITLE
CMP0074 policy for exported config

### ISSUE THREAD

**Issue by cyrush** [LLNL/conduit#633]
this is fixed in conduit's build env, but not in downstream envs that are loading conduit via exported cmake targets. 

```

  Policy CMP0074 is not set: find_package uses <PackageName>_ROOT variables.
  Run "cmake --help-policy CMP0074" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  CMake variable HDF5_ROOT is set to:

    /Users/harrison37/Work/spack/opt/spack/darwin-mojave-x86_64/apple-clang-11.0.0/hdf5-1.10.7-mtlqb7bltpwsikbcxdbjj5skaovryy7z

  Environment variable HDF5_ROOT is set to:

    /Users/harrison37/Work/spack/opt/spack/darwin-mojave-x86_64/apple-clang-11.0.0/hdf5-1.10.7-mtlqb7bltpwsikbcxdbjj5skaovryy7z/bin
```

I believe @ax3l reported this as well.

**Comment by cyrush** [comment]
resolved in #637

**Comment by cyrush** [comment]
Note, we will also see these warnings with MPI on some systems that set the MPI_ROOT env var as part of modules.
